src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                 skos:prefLabel "lithodeme"@en .
+
+
+###  http://sweetontology.net/realmGeol/LithodemicFeature
+soreag:LithodemicFeature rdf:type owl:Class ;
+                         rdfs:subClassOf soreag:GeologicFeature ;
+                         owl:disjointWith soreag:StratigraphicFeature ;
+                         dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                         dcterms:source <https://doi.org/10.1306/05230505015> ;
+                         skos:definition "A Lithodemic Unit is a defined body of predominantly intrusive, highly metamorphosed, or intensely deformed rock that, because it is intrusive or has lost primary structure through metamorphism or tectonism, generally does not conform to the Law of Superposition. Expressions of age of lithodemic units (because they are non-stratigraphic) should be given in terms of the appropriate geochronologic (geological-time) unit, and thus not by chronstratigraphic (time-rock) terms. (North American Stratigraphic Code, 2005)"@en ;
+                         skos:prefLabel "lithodemic feature"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Lithosphere
 soreag:Lithosphere rdf:type owl:Class ;
-                 rdfs:subClassOf soreag:GeosphereLayer ,
-                                 [ rdf:type owl:Restriction ;
-                                   owl:onProperty sorelsc:containsMaterial ;
-                                   owl:allValuesFrom somarock:Rock
-                                 ] ;
-                 rdfs:label "lithosphere"@en .
+                   rdfs:subClassOf soreag:GeosphereLayer ,
+                                   [ rdf:type owl:Restriction ;
+                                     owl:onProperty sorelsc:containsMaterial ;
+                                     owl:allValuesFrom somarock:Rock
+                                   ] ;
+                   rdfs:label "lithosphere"@en .
+
+
+###  http://sweetontology.net/realmGeol/LithostratigraphicUnit
+soreag:LithostratigraphicUnit rdf:type owl:Class ;
+                              rdfs:subClassOf soreag:StratigraphicUnit ;
+                              dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                              dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+                              skos:definition "Lithostratigraphic units are bodies of rocks, bedded or unbedded, that are defined and characterized on the basis of their lithologic properties and their stratigraphic relations. Lithostratigraphic units are the basic units of geologic mapping. (ICS, Online, December 2019)"@en ;
```

Comment:
A geologic unit defined on the basis of observable and distinctive lithologic properties or a combination of lithologic properties and stratigraphic relationships. Denotes a particular body of rock.  (Salvador, A., 1994, International Stratigraphic Guide, Second Edition)

Note--allostratigraphic and sequence stratigraphic units are defined by their bounding surfaces; these are commonly used for mapping but I don't think they are technically lithostratigraphic units.